tests/test_obslog_command.rs

```diff
@@ -33,14 +33,27 @@ fn test_obslog_with_or_without_diff() {
     insta::assert_snapshot!(stdout, @r###"
     @  rlvkpnrzqnoo test.user@example.com 2001-02-03 04:05:10.000 +07:00 66b42ad36073
     │  my description
-    ◉  rlvkpnrzqnoo test.user@example.com 2001-02-03 04:05:09.000 +07:00 af536e5af67e conflict
+    ◉  rlvkpnrzqnoo (obs) test.user@example.com 2001-02-03 04:05:09.000 +07:00 af536e5af67e conflict
```

Comment:
`(obs)` is unclear to me, but I feel `(obsolete)` would be too verbose (especially in obslog), so I have no better idea.

Just fyi, in Mercurial, obsolete commit is represented as `x` in graph.

Reply:
I feel like "hidden" is better than "obsolete". That would match `visible_heads()` (which I recently renamed `heads()` to).

And yes, it might be a good idea to indicate them in the graph. Even if we choose to present it only there by default, it's probably good to also have the information in the template so users can access it even when using `--no-graph`.

Reply:
Yet again bringing up git-branchless, such commits are marked with × in place of ◉ in it

Reply:
> (obs) is unclear to me, but I feel (obsolete) would be too verbose

I think I'll go with "hidden" for now, as Martin suggested, since it's short. I don't like that word, but Sapling and git-branchless use it and I don't have a better idea at the moment.

> Yet again bringing up git-branchless, such commits are marked with × in place of ◉ in it

Agreed, this would be great. It doesn't seem like a trivial feature to add at the moment AFAIK because of how the graph is implemented, but it could certainly be done.

Reply:
> Agreed, this would be great. It doesn't seem like a trivial feature to add at the moment AFAIK because of how the graph is implemented, but it could certainly be done.

By the way, I'm soon going to work on defining a revset for immutable commits, and another revset for getting shorter commit/change id prefixes. That work might make it easier to also check if a given commit is visible.

Reply:
Yeah, it would make a lot of sense to use commit markers to distinguish between immutable, mutable, and obsolete commits. They could be, for example, `◆, ◉, ×`. Another nice symbol is `■`. I'm mostly taking these from https://www.w3.org/TR/xml-entity-names/025.html